tests/datasets/test_datasets.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Data61, CSIRO
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from stellargraph.datasets import download_all_datasets, Cora
+from urllib.error import URLError
+
+
+@pytest.mark.slow
```

Comment:
Do we have anything that activates or deactivates these sort of tests?

Reply:
no, this was a mistake (undefined so gave warnings) - have removed.